frontend/src/models/player.ts

```diff
@@ -172,10 +172,28 @@ export class Player {
     notifier.info("player-announced-" + announcement, { name: this.name });
   }
 
-  hasAnnounced(...announcements: Announcement[]) {
+  hasAnnounced(...announcements: Announcement[]): boolean {
```

Comment:
just by looking at the method signature, I find it impossible to distinguish between `hasAnnounced(announcements): boolean` and `wasAnnounced(announcements): boolean`. Maybe moving it to `party.ts` helps, but maybe this needs some clearer naming on top.

Reply:
You're right.
`hasAnnounced()` has been removed, since it wasn't used anyway - don't know why I didn't notice 😭
I am using `hasPartyAnnounced()` instead of `wasAnnounced()` now, so it's only `hasPartyAnnounced(announcements): boolean` left over which should be clear enough 👍🏻